sympy/physics/control/tests/test_lti.py

```diff
@@ -0,0 +1,258 @@
+    ssm = StateSpaceModel(TransferFunctionModel(G))
+    assert ssm.represent[0] == Matrix([[-9 / 2, 0, -6, 0, -2, 0],
+                                       [0, -9 / 2, 0, -6, 0, -2],
+                                       [1, 0, 0, 0, 0, 0],
+                                       [0, 1, 0, 0, 0, 0],
+                                       [0, 0, 1, 0, 0, 0],
+                                       [0, 0, 0, 1, 0, 0]])
+    assert ssm.represent[1] == Matrix([[1, 0],
+                                       [0, 1],
+                                       [0, 0],
+                                       [0, 0],
+                                       [0, 0],
+                                       [0, 0]])
+    assert ssm.represent[2] == Matrix([[-6, 3, -24, 15 / 2, -24, 3],
+                                       [0, 1, 1 / 2, 3 / 2, 1, 1 / 2]])
+    assert ssm.represent[3] == Matrix([[2, 0],
+                                       [0, 0]])
+
+
+def test_TransferFunctionModel_create():
+    s = symbols('s')
+    G = Matrix([1 / s, 1 / (s + 1)])
+    tfm = TransferFunctionModel(G)
+    assert tfm.G == G
+
+    g = symbols('g')
+    G = Matrix([[s / (s**2 + 3), 1 / s],
+                [1 / (s + g), s / (s**2 - g * s - 3)]])
+    tfm = TransferFunctionModel(G)
+    assert tfm.G == G
+
+    A = Matrix([[-R(9, 2), 0, -6, 0, -2, 0],
+                [0, -R(9, 2), 0, -6, 0, -2],
+                [1, 0, 0, 0, 0, 0],
+                [0, 1, 0, 0, 0, 0],
+                [0, 0, 1, 0, 0, 0],
+                [0, 0, 0, 1, 0, 0]])
+    B = Matrix([[1, 0],
+                [0, 1],
+                [0, 0],
+                [0, 0],
+                [0, 0],
+                [0, 0]])
+    C = Matrix([[-6, 3, -24, R(15, 2), -24, 3],
+                [0, 1, R(1, 2), R(3, 2), 1, R(1, 2)]])
+    D = Matrix([[2, 0],
+                [0, 0]])
+    tfm = TransferFunctionModel(StateSpaceModel(A, B, C, D), s)
+    G = Matrix(
+        [[(4 * s - 10) / (2 * s + 1), 3 / (s + 2)],
+         [1 / (2 * s**2 + 5 * s + 2), (s + 1) / (s**2 + 4 * s + 4)]])
+    assert simplify(tfm.G - G) == zeros(2, 2)
+
+
+def test_StateSpaceModel_eval():
+    t, y1, y2, omega = symbols('t, Y1, Y2, omega')
+    ssm = StateSpaceModel(eye(2) * omega, zeros(2, 1),
+                          eye(2), zeros(2, 1))
+    u = eye(1) * omega
+    x0 = Matrix([y1, y2])
+    assert ssm.evaluate(u, x0, t) == Matrix([y1 * exp(omega * t),
+                                             y2 * exp(omega * t)])
+
+    ssm = ssm.subs(omega, 1)
+    u = u.subs(omega, 1)
+    x0 = x0.subs([(y1, 1.0), (y2, 0.0)])
+    sol_num = ssm.evaluate(u, x0, (t, [0, 1.0]))
+    assert sol_num == [Matrix([1.0, 0.0]), Matrix([e, 0.0])]
+
+    A = Matrix([[-1, 1], [1, -1]])
+    ssm = StateSpaceModel(A, ones(2, 1),
+                          eye(2), zeros(2, 1))
+    u = Matrix([exp(2 * t)])
+    x0 = Matrix([1, 0])
+    sol = ssm.evaluate(u, x0, t, do_integrals=True, simplify=True)
```

Comment:
Would `solve()` be a better name? This seems to solve the differential equation, no?

Reply:
YES, I think one `solve` method each for both classes should serve the purpose for symbolic solutions.